ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -144,12 +144,21 @@ private void onBlockAdded(final BlockAddedEvent blockAddedEvent, final Blockchai
     }
   }
 
-  private void handleNewBlockFromNetwork(final EthMessage message) {
+  void broadcastBlock(final Block block, final UInt256 difficulty) {
+    final List<EthPeer> availablePeers =
+        ethContext.getEthPeers().availablePeers().collect(Collectors.toList());
+    for (EthPeer ethPeer : availablePeers) {
+      ethPeer.propagateBlock(block, difficulty);
+    }
+  }
```

Comment:
If you're feeling adventurous you could merge the collection into a list and forLoop into a single line: 
`ethContext.getEthPeers().availablePeers().forEach(ethPeer -> ethPeer.propagateBlock(block, difficulty));`